models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -391,6 +391,7 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
       
       # reconstruct total wood C
       wood_total_C <- IC$AbvGrndWood / IC$abvGrndWoodFrac
+      if (is.na(wood_total_C) | is.infinite(wood_total_C) | is.nan(wood_total_C) | wood_total_C <0) wood_total_C <- 0
```

Comment:
Recall my earlier comment about single (`|`) vs double (`||`) subsetting. If `wood_total_C` is a scalar, just change all of the `|` to `||`. But if it's a vector, you'll want to wrap the whole thing in `any` or `all`, as appropriate.

Reply:
I don't get this comment. Can you explain. if I do doubles they will fail. is that what the any/all command is for?

Reply:
nm. Shawn told me it's on the call_modis PR

Reply:
@bailsofhay https://github.com/PecanProject/pecan/pull/2338#discussion_r277086316